portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Uses UserService to login the current user and foward them to a desired page
+ */
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+  private static final UserService userService = UserServiceFactory.getUserService();
```

Comment:
nit: What do you think about adding a newline between this member variable and the next function, just to give it a little space and call it out (this file and similar for the other one).

Reply:
Yeah I probably left out a line here by mistake, I'll go in and fix it.